dicetables/factory/errorhandler.py

```diff
@@ -1,5 +1,3 @@
-
-
 class EventsFactoryError(AssertionError):
```

Comment:
check this file!

Reply:
not a fan of this file.  this is an improvement.  there may be some code checking required later.  i'll make an issue about it.